### DISCUSSION TITLE
Flutter AbsorbPointer vs IgnorePointer difference

### DISCUSSION THREAD

**Discussion by cahkas** [community/community#29922] [Mobile] [answered]
Flutter AbsorbPointer vs IgnorePointer difference

**Comment by surajadkhari** [comment, marked as answer]
AbsorbPointer prevents its subtree from receiving pointer events by terminating hit testing at itself.

IgnorePointer, which also prevents its children from receiving pointer events but is itself invisible to hit testing.
